manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.de.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: fjsoft.MyPhoneExplorer
+PackageVersion: 1.8.15
+PackageLocale: de
+Publisher: F.J. Wechselberger
+PackageName: MyPhoneExplorer Desktop

```

```suggestion
PackageName: MyPhoneExplorer
```
It shows as MyPhoneExplorer in Control Panel.